.readthedocs.yaml

```diff
@@ -8,6 +8,17 @@ build:
   os: ubuntu-22.04
   tools:
     python: "3.12"
+  jobs:
+    post_checkout:
+      # https://docs.readthedocs.io/en/stable/build-customization.html#support-git-lfs-large-file-storage
+      - wget https://github.com/git-lfs/git-lfs/releases/download/v3.1.4/git-lfs-linux-amd64-v3.1.4.tar.gz
+      - tar xvfz git-lfs-linux-amd64-v3.1.4.tar.gz
+      - git config filter.lfs.process "`pwd`/git-lfs filter-process"
+      - git config filter.lfs.smudge  "`pwd`/git-lfs smudge -- %f"
+      - git config filter.lfs.clean "`pwd`/git-lfs clean -- %f"
+      - ./git-lfs install
+      - ./git-lfs fetch
+      - ./git-lfs checkout
```

Comment:
Interesting, we've never bothered with LFS in Wagtail's own docs 😆

Reply:
Yeah I'm a bit paranoid about repositories growing out of control, and LFS is such an elegant solution. I'm kind of bummed that many SAAS platforms _don't_ support LFS out of the box though.